pkg/kernel/tools/heal/liveness_check.go

```diff
@@ -35,6 +36,11 @@ const (
 // NewKernelLivenessCheck is an implementation of heal.LivenessCheck. It sends ICMP
 // ping and checks reply. Returns false if didn't get reply.
 func NewKernelLivenessCheck(deadlineCtx context.Context, conn *networkservice.Connection) bool {
```

Comment:
Why do we have `new` prefix when the function doesnt supply anything?
```suggestion
func KernelLivenessCheck(deadlineCtx context.Context, conn *networkservice.Connection) bool {
```